Yosemite/YosemiteTests/Stores/AppSettingsStoreTests.swift

```diff
@@ -877,6 +877,24 @@ extension AppSettingsStoreTests {
         XCTAssert(Calendar.current.isDate(actualRemindAfter, inSameDayAs: twoWeeksTime))
     }
 
+    func test_setFeatureAnnouncementDismissed_with_remindAfter_seven_days_stores_reminder_date_in_one_week() throws {
```

Comment:
It would be great to use the agreed test function naming syntax here as well:

`test_setFeatureAnnouncementDismissed_when_remindAfter_is_seven_days_stores_reminder_then_date_saved_date_is_one_week`

Reply:
Yes, you're right! Changed f8a2586b813a1a49f3ff0d4f6943c3aa82661b28